frontend/src/components/profile/header/ProfileMemberHeader.tsx

```diff
@@ -31,9 +31,20 @@ export const ProfileMemberHeader = ({
             {personnel.firstName} {personnel.lastName}
           </h2>
           {roles?.includes(Role.COORDINATOR) && (
-            <span>
-              <PersonnelStatus status={personnel?.status} />
-            </span>
+            <div className="flex flex-col space-y-2">
```

Comment:
I think line 33 needs to be removed (my bad!) and this needs to be flex-row? In the design, active in BOTH only says "ACTIVE" but I'm okay with doing it this way for now

Reply:
Oh good catch - and yeah I saw that re: if active in both just show one, but it seemed kinda silly to me lol. I will change it though.